src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -146,7 +146,7 @@ export class NativeDateAdapter extends DateAdapter<Date> {
   }
 
   clone(date: Date): Date {
-    return this.createDate(this.getYear(date), this.getMonth(date), this.getDate(date));
+    return new Date(date);
```

Comment:
For some reason in google3 it complains about `Argument of type 'Date' is not assignable to parameter of type 'string | number'`. Instead maybe you can do `new Date(date.getTime())`

Reply:
Done.